### PR TITLE
Create portfolio-section.html, portfolio.css

### PR DESCRIPTION
**Pull request by duylong13**

### CODE REVIEW

File: Practice2/src/partials/portfolio-section.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        <source media="(min-width: 1440px)" srcset="./assets/images/valley-desktop.jpg">
+        <source media="(min-width: 768px)" srcset="./assets/images/valley-tablet.jpg">
+        <source media="(min-width: 320px)" srcset="./assets/images/valley-mobile.jpg">
+        <img src="./assets/images/valley-mobile.jpg" alt="valley" class="maximum-size-img">
+      </picture>
+    </div>
+  </div>
+  <div>
+    <ul class="list">
+      <i class="arrow left"></i></p>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/portfolio-section.html]: Please check why </p> here

File: Practice2/src/partials/portfolio-section.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      <li class="number-list number-list-one">1</li>
+      <li class="number-list number-list-two">2</li>
+      <li class="number-list number-list-three">3</li>
+      <li class="number-list number-list-four">4</li>
+      <li class="number-list number-list-five">5</li>
+      <i class="arrow right"></i></p>
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/portfolio-section.html]: Please check why </p> here

File: Practice2/src/partials/portfolio-section.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        <source media="(min-width: 320px)" srcset="./assets/images/sea-mobile.jpg">
+        <img src="./assets/images/sea-mobile.jpg" alt="sea" class="maximum-size-img">
+      </picture>
+    </div>
+    <div class="portfolio-item portfolio-item-six">
+      <div class="item-six-common">
+      <p class="item-six-sub-heading">Scenery Photos</p>
+      <p class="item-six-sub-sub-heading">By Evgeny Metilan</p>
+      <div class="item-six-underline"></div>
+      <h5 class="item-six-main-heading">Wild
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/portfolio-section.html]: Don't hard code by enter. You should use <br>

File: Practice2/src/partials/portfolio-section.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        <source media="(min-width: 320px)" srcset="./assets/images/blue-mountain-mobile.jpg">
+        <img src="./assets/images/blue-mountain-mobile.jpg" alt="blue-mountain" class="maximum-size-img">
+      </picture>
+      <div class="blue-mountain-common">
+        <h5 class="blue-mountain-heading">Blue Mountains</h5>
+        <p class="blue-mountain-text">17 Apr 2016</p>
+        <div class="blue-mountain-underline"></div>
+        <p class="blue-mountain-sub-text">Planning a vacation to Europe? No doubt you have certain expectations about
+          your upcoming journey.</p>
+        <div class="parameter">
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/portfolio-section.html]: Using ul, li in this case

**Review by vuongnguyen-agilityio — CHANGES_REQUESTED** [review]

File: Practice2/src/partials/portfolio-section.html (hunk 2 above)
Comment (duylong13) [reply Practice2/src/partials/portfolio-section.html]: Fixed

File: Practice2/src/partials/portfolio-section.html (hunk 3 above)
Comment (duylong13) [reply Practice2/src/partials/portfolio-section.html]: Fixed

File: Practice2/src/partials/portfolio-section.html (hunk 1 above)
Comment (duylong13) [reply Practice2/src/partials/portfolio-section.html]: Fixed

File: Practice2/src/partials/portfolio-section.html (hunk 4 above)
Comment (duylong13) [reply Practice2/src/partials/portfolio-section.html]: Fixed

**Pull request merged**